projects/plugins/jetpack/_inc/client/components/settings-group/test/component.js

```diff
@@ -35,7 +35,6 @@ describe( 'SettingsGroup', () => {
 			'notes',
 			'subscriptions',
 			'protect',
-			'enhanced-distribution',
```

Comment:
I think there is one more instance like that in `projects/plugins/jetpack/_inc/client/components/settings-card/test/component.js`.

Reply:
Can you double-check? I don't see it referenced in `settings-card/test/component.js`

Reply:
It would help if I saved the file before committing. bc1128acba